reactor-core/src/main/java/reactor/util/retry/Retry.java

```diff
@@ -98,24 +111,47 @@ default RetrySignal copy() {
 	 *
 	 * @param maxAttempts the maximum number of retry attempts to allow
 	 * @param minBackoff the minimum {@link Duration} for the first backoff
-	 * @return the builder for further configuration
+	 * @return the exponential backoff spec for further configuration
 	 * @see RetryBackoffSpec#maxAttempts(long)
 	 * @see RetryBackoffSpec#minBackoff(Duration)
 	 */
-	static RetryBackoffSpec backoff(long maxAttempts, Duration minBackoff) {
+	//FIXME marble diagram
+	public static RetryBackoffSpec backoff(long maxAttempts, Duration minBackoff) {
 		return new RetryBackoffSpec(maxAttempts, t -> true, false, minBackoff, MAX_BACKOFF, 0.5d, Schedulers.parallel(),
 				NO_OP_CONSUMER, NO_OP_CONSUMER, NO_OP_BIFUNCTION, NO_OP_BIFUNCTION,
 				RetryBackoffSpec.BACKOFF_EXCEPTION_GENERATOR);
 	}
 
+	/**
+	 * A {@link RetryBackoffSpec} preconfigured for fixed delays (min backoff equals max backoff, no jitter), given a maximum number of retry attempts
+	 * and the fixed {@link Duration} for the backoff.
+	 * <p>
+	 * Note that calling {@link RetryBackoffSpec#minBackoff(Duration)} or {@link RetryBackoffSpec#maxBackoff(Duration)} would switch
+	 * back to an exponential backoff strategy.
+	 *
+	 * @param maxAttempts the maximum number of retry attempts to allow
+	 * @param fixedDelay the {@link Duration} of the fixed delays
+	 * @return the fixed delays spec for further configuration
+	 * @see RetryBackoffSpec#maxAttempts(long)
+	 * @see RetryBackoffSpec#minBackoff(Duration)
+	 * @see RetryBackoffSpec#maxBackoff(Duration)
+	 */
+	//FIXME marble diagram
+	public static RetryBackoffSpec fixedDelays(long maxAttempts, Duration fixedDelay) {
```

Comment:
```suggestion
	public static RetryBackoffSpec fixedDelay(long maxAttempts, Duration fixedDelay) {
```

Also, it would be nice to have one without `maxAttempts`

Reply:
all of these start from `maxAttempts` though, except `immediately()`... I wonder about that last one tbh, and about whether or not we should encourage infinite attempts at all (even though the specs make it possible via `maxAttempts(Long.MAX_VALUE)` in effect)

Reply:
I see no issue with infinite attempts, given that spec supports cancellation, and there can and most probably will be `.timeout` downstream